libpod/image/image.go

```diff
@@ -1475,7 +1476,8 @@ func (i *Image) Save(ctx context.Context, source, format, output string, moreTag
 			return errors.Wrapf(err, "error getting OCI archive ImageReference for (%q, %q)", output, destImageName)
 		}
 	case "oci-dir":
-		destRef, err = directory.NewReference(output)
+		destImageName := imageNameForSaveDestination(i, source)
+		destRef, err = layout.NewReference(output, destImageName)
```

Comment:
(Preserving the `// destImageName may be ""` comment would be consistent, and not lead the readers to wonder why it can be so for one and not the other.)

Reply:
Fixed

Reply:
I don’t see any change